### PR TITLE
Indicate muted streams. Hide muted streams/topics

### PR DESCRIPTION
**Pull request by borisyankov**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@borisyankov, it looks like you've already signed the Dropbox CLA. Thanks!

File: src/mute/muteReducers.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {
+  REALM_INIT,
+} from '../constants';
+
+const initialState = [];
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case REALM_INIT:
```
Comment (neerajwahi) [inline src/mute/muteReducers.js]: Where is the mute reducer used? It doesn't look like this has any effect

Comment (borisyankov) [inline src/mute/muteReducers.js]: MainScreenContainer.js >state.mute

File: src/autocomplete/EmojiAutocomplete.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default class EmojiAutocomplete extends Component {
 
   render() {
     const { filter, onAutocomplete } = this.props;
-    const emojis = getFilteredEmojiList(filter).slice(1, 5);
+    const emojis = getFilteredEmojiList(filter).slice(0, 5);
```
Comment (borisyankov) [inline src/autocomplete/EmojiAutocomplete.js]: Will add comment. (equals 'Limit to at most 5 results')

File: src/mute/muteReducers.js (hunk 1 above)
Comment (neerajwahi) [inline src/mute/muteReducers.js]: 👍

Comment (neerajwahi) [inline src/mute/muteReducers.js]: I'm not sure this should be its own reducer though (maybe something like `state.settings` ...?) or at least we might want to rename it `state.mutedTopics` to be clearer

**Comment by neerajwahi** [comment]
Looks good overall! Going to merge and we can think about renaming `state.mute` later

**Pull request merged**